DEPENDENCIES

```diff
@@ -19,7 +19,7 @@ Prerequisites for building the OpenDataPlane (ODP) API
 
 3. Required libraries
 
-   Libraries currently required to link: openssl, libatomic
+   Libraries currently required to link: openssl, libatomic, libconfig
```

Comment:
Does this have to be a hard dependency? Can we have this feature be omitted (hardcoded defaults are used) if libconfig is not available?

Reply:
As we already require for example autoconf, libtool, and pkg-config, adding libconfig dependency shouldn't be a big issue. However, if this causes problems we can remove the hard dependency by wrapping all used libconfig functions with dummy implementations. I would prefer not doing this if not explicitly required.